felix/fv/xdp_test.go

```diff
@@ -99,13 +110,15 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ XDP tests with initialized
 			Expect(err).NotTo(HaveOccurred())
```

Comment:
Better to wait for hostEndpoints to be created.

Reply:
yes, the synchronization here is not great, I am working on fixing this in a separate PR, thanks for the tip!